scripts/preprocess/process_zip_to_shp.R

```diff
@@ -15,6 +15,10 @@ read_shp_zip <- function(zipfile) {
   # read the layer from the shapefile
   data_out <- rgdal::readOGR(shp_path, layer=layer, verbose=FALSE)
   
+  # filter out Northern Mariana Island (STATEFP=='69', ABBV='MP'), Guam
+  # (STATEFP=='66', ABBV='GU'), and American Samoa (STATEFP=='60', ABBV='AS')
```

Comment:
are these something we will want to explicitly state on the viz somewhere?

Reply:
like a footnote?

Reply:
i don't think so...we just don't have water use data for those territories. lots of vizzies get presented without justifying this omission (and even omit other places, e.g., PR and VI, without saying anything)